components/ILIAS/Contact/classes/class.MailingListsMembersTable.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+use ILIAS\Data;
+use ILIAS\UI;
+use Psr\Http\Message\ServerRequestInterface;
+
+class MailingListsMembersTable implements UI\Component\Table\DataRetrieval
```

Comment:
Can you please rename the file to `MailingListsMembersTable.php`. Filename (without extension) and the class name SHOULD always be identical.

Maybe it is worth to create a new sub-folder "MailingLists" and move the file to this folder. Then, we should introduce a `namespace ILIAS\Contact\MailingLists`.